### ISSUE TITLE
Notes panel shrinks width as object highlighting changes 

### ISSUE THREAD

**Issue by emyoulation** [cdhorn/profileview#67]
I was in the Example Gramps (with the active person set to Dr. Lewis Anderson, Sr "Big Louie" Garner von Zieliński, Sr )
My Profile configuration has all 7 panels active (Associations were off-screen.)

Every time I refreshed (by rolled over/off a hyperlink), the highlighting would correctly invert but the Note panel width shrunk a few pixels and the text re-flowed.

It did one change at at time if the rollover was in a panel to the left of the Note panel.  But rolling over an object to the right (like over a media item thumbnail) repeated the refresh after each text reflow.

The panel would shrank 16 times before hitting its minimum width. Switching people and back would reset and repeat the shrinkage. (Might be another Windoze CSS thing!)

**Comment by cdhorn** [comment]
Yes this is a problem for me as well, Fedora 34.

**Comment by emyoulation** [comment]
Started from the command line to see if any error appeared. None for the shrinking Note panel.

But it complained when I collapse a panel:

**Comment by cdhorn** [comment]
Maybe result of removing the widget completely, not sure.

Some of the resizing and layout issues are tough. My understanding is you're supposed to let Gtk handle all that and the use of fixed sizes is discouraged.

**Comment by emyoulation** [comment]
Can you tell me how to restore a Panel that has been collapsed when the "Hide" option is enabled?

I have been changing Active Person to reset the panel visibility.

**Comment by cdhorn** [comment]
To actually "free" the space it occupied for the other groups to use it asks the parent to remove it. So yes, you would need to select active person to trigger a redraw as it no longer exists.

**Comment by emyoulation** [comment]
You probably noticed that the collapsing of a panel can resizing the application window.  It drags the bottombar up. 

This seems to be related to the Associations panel and stacking.  Possibly the lack of Contextual pop-up menus for the Associations panel.

Try stacking the Associations on top of the Timeline panel (no hiding) 
![image](https://user-images.githubusercontent.com/69127217/127403061-f60b07d2-6881-4670-a699-9bda9ae40cd7.png)

Make sure Gramps is not Maximized. 
Add a couple Associations to the Home Person

In the Relationships view Profile mode, toggle the collapse on the Associations a couple times.  The whole gramps Window resizes vertically.  

(Be careful if you drag resize the bottombar. There's a bug ... in ALL Gramps views... where the bottombar can be resized past where it can be grabbed to resize again.  You can resize it out of existence.)

**Comment by cdhorn** [comment]
Commit dca5a79f76a95b36ff7b98ec1325c785bf240048 sets a minimum note frame size when it is in a group and that seems to help a great deal for me at least.

**Comment by cdhorn** [comment]
Just added a scrolled mode as well, when enabled in full page mode it enables scrolling on the individual groups.

**Comment by cdhorn** [comment]
Stacking thing may be fixed now as well.

**Comment by emyoulation** [comment]
It is necessary to step back from the Profile Gramplet for a bit. Although not likely to be related to your add-on, I've encountered some critical issues that must be isolated.

Since updating to 5.1.4, I've experience 4 spontaneous exits. And there are 2 categories of serious intermittent performance issues (slow populating of the Select Place dialog & incomplete View category switching) and one that is constant (Filter gramplet slowdown).  

So I've removed your Profile view mode & Nick's Combined view mode while doing isolation.

The spontaneous exits are bone chilling... but the intermittents are much more persistently aggravating.
